modules/terraform-aci-route-control-route-map/main.tf

```diff
@@ -15,7 +15,7 @@ resource "aci_rest_managed" "rtctrlProfile" {
   content = {
     name  = var.name
     descr = var.description
-    type  = "combinable"
```

Comment:
I believe this will be a breaking change since it is moving from a hard-coded value to a variable.